survey_percent_question/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
Please add the target attribute for the image, as per the [readme template](https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)